test/ufuzz.js

```diff
@@ -585,6 +585,14 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
     }
 }
 
+function createPropAccess(recurmax, stmtDepth, canThrow) {
+    var expression;
+    do {
+        expression = createExpression(recurmax, COMMA_OK, stmtDepth, canThrow);
+    } while (~expression.indexOf("[]"));
```

Comment:
Not crazy about this. My patch is more efficient. Don't waste cycles generating stuff you don't need.